chatbot-core/backend/app/models/api.py

```diff
@@ -1,10 +1,11 @@
 from pydantic import BaseModel
 from typing import Optional, Any
 
-from utils.error_handler import ErrorCodesMappingNumber
+from app.utils.error_handler import ErrorCodesMappingNumber
 
 
 class APIResponse(BaseModel):
+    status_code: int = 200
```

Comment:
Don't need this, you can remove it. I will include `status_code` in the parameter of router (e.g. `router.post("/", status_code=status.HTTP_201_SUCCESSFULLY)`

Reply:
Ok.